apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -124,6 +133,15 @@ export class ApplicationService {
       await Promise.all(promises)
     }
 
+    await this.sendEmail(
+      {
+        name: user.name,
+        address: application.email,
+      },
+      appModel.id,
```

Comment:
Is there any reason why we are sometimes using the appModel and sometimes the application?